_python/main/urls.py

```diff
@@ -67,4 +68,9 @@ def to_url(self, value):
     path('users/<int:user_id>', views.dashboard, name='dashboard'),
     path('casebooks/<idslug:casebook_param>/sections/<ordslug:ordinals_param>/', views.section, name='section'),
     path('casebooks/<idslug:casebook_param>/', views.casebook, name='casebook'),
+    re_path(r'^about/?$', DirectTemplateView.as_view(template_name='pages/about.html'), name='about'),
+    re_path(r'^privacy-policy/?$', DirectTemplateView.as_view(template_name='pages/privacy-policy.html'), name='privacy-policy'),
+    re_path(r'^terms-of-service/?$', DirectTemplateView.as_view(template_name='pages/terms-of-service.html'), name='terms-of-service'),
+    re_path(r'^faq/?$', DirectTemplateView.as_view(template_name='pages/about.html'), name='faq'),
```

Comment:
It doesn't really matter, but I don't think we need the more-advanced "re_path" here; regular path with "about/", "privacy-policy" etc. ought to do the job.